.gitbook.yaml

```diff
@@ -0,0 +1,5 @@
+root: ./docs
```

Comment:
Gitbook config - other than this file, gitbook will only ever modify contents in the docs/ folder (since that is the defined root)